i18n/de_DE.csv

```diff
@@ -55,3 +55,5 @@ SWIFT,SWIFT
 "CMS Page for Shipping Info","CMS-Seite für Versandkosten"
 "Show ""incl. Shipping Cost"" instead of ""excl. Shipping Cost""","Zeige ""inkl. Versandkosten"" anstatt ""exkl. Versandkosten"""
 "Display Delivery Time on Product Listing","Lieferzeit auf Produktübersichtsseiten anzeigen"
+"No dynamic shipping tax calculation","Keine dynamische Versandsteuerberechnung"
```

Comment:
Please re-generate the English language file and then edit the German file. Hint:

    n98-magerun2.phar i18n:collect-phrases -o app/code/FireGento/MageSetup/i18n/en_US.csv app/code/FireGento/MageSetup/

Reply:
It seems that you re-generated the English file. Great! Now IMHO, we need to update the German file accordingly (add the strings to the position similar to the EN file, not at the end of the file).